identity-wallet/src/state/reducers/dev_mode.rs

```diff
@@ -246,23 +233,23 @@ async fn load_credential_images(
 
     // Credentials
     let mut image_bytes: &[u8] = include_bytes!("../../../resources/images/cuddlyferris.svg");
-    let file_name = format!("{}.svg", personal_information_id);
+    let file_name = format!("{}.svg", PERSONAL_INFORMATION.clone().display_credential.id);
```

Comment:
PERSONAL_INFORMATION**.clone()**.display_credential.id

these don't need to be cloned, same goes for the other credentials :)